### PR TITLE
Provide System.Composition.Convention package readme

### PR DESCRIPTION
**Pull request by eNeRGy164**
Fill out details for the `System.Composition.Convention` NuGet package readme.

See https://github.com/dotnet/runtime/issues/99358 and https://github.com/dotnet/runtime/pull/106443

### CODE REVIEW

**Review by ericstj — COMMENTED** [review]
LGTM, @steveharter @buyaa-n?

**Review by ericstj — APPROVED** [review]

File: src/libraries/System.Composition.Convention/src/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+`System.Composition.Convention` is part of the Managed Extensibility Framework (MEF) 2.0, a composition library for .NET that enables dependency injection through attributes.
```
Comment (ericstj) [inline src/libraries/System.Composition.Convention/src/PACKAGE.md:5]: Actually this one is awkward to mention attributes here.  This demonstrates usage without attributes.  Should we update all of these to be more broad to indicate the other means of injection? 
```suggestion
`System.Composition.Convention` is part of the Managed Extensibility Framework (MEF) 2.0, a composition library for .NET that enables dependency injection through attributes or conventions.
```

Comment (eNeRGy164) [reply src/libraries/System.Composition.Convention/src/PACKAGE.md:5]: Ah, that might be better indeed

**Review by buyaa-n — APPROVED** [review]
LGTM, thank you!

**Pull request merged**